fastparse/shared/src/main/scala/fastparse/sequence/VectorApi.scala

```diff
@@ -0,0 +1,78 @@
+package fastparse.sequence
+
+import fastparse.core.{ ParserApi, ParserApiImpl }
+import fastparse.parsers.Terminals.AnyElems
+import scala.reflect.ClassTag
+import fastparse.{ core, parsers }
+
+class VectorApi[A]()(implicit evidence: ClassTag[A]) extends SeqApi[A, Vector[A]](implicitly, VectorReprOps[A]) {
+  val AnyElem = parsers.Terminals.AnyElem[A, Vector[A]]("AnyElem")
+  def AnyElem(count: Int) = AnyElems[A, Vector[A]]("AnyElems", count)
+  def ElemPred(pred: A => Boolean): P0 = SeqItemParsers.Intrinsics.ElemPred[A, Vector[A]]("ElemPred", pred)
+  def ElemIn(items: Seq[A]*): P0 = SeqItemParsers.Intrinsics.ElemIn[A, Vector[A]]("ElemIn", items: _*)
+  def ElemsWhile(pred: A => Boolean, min: Int = 1) = SeqItemParsers.Intrinsics.ElemsWhile[A, Vector[A]]("ElemsWhile", pred, min)
+}
+
+/**
+ * This class is used to parse a Vector of items
```

Comment:
Rather than including this as a doc-comment, let's include it in the generated doc-site as a separate section, below `Byte Parsers`: https://github.com/lihaoyi/fastparse/blob/master/readme/Readme.scalatex#L24

Perhaps we could call it `Token Parsers` or similar. That should then include code examples extracted from your unit tests, so e.g. instead of copy-pasting the example into the docs we'd tidy up some of the examples you already have in `SequenceTests.scala` and embed them using `@hl.ref`.

We should follow the rough layout of the existing `Byte Parsers` section (http://www.lihaoyi.com/fastparse/#ByteParsers):

- Simple, motivating example

- Any operators that are specific to token parsing (e.g. `F{}`)

- A few of the token parsers that are identical to the string parsers. No need to have as exhaustive a list as http://www.lihaoyi.com/fastparse/#EquivalentByteParsers, but we should have at least some comparisons

- At least one slightly-meatier, more-realistic token parser to better motivate what these token parsers are used for, and provide an example for how the parsers look like when they grow beyond trivial cases.